updated IdP note (#14364)
docker/docker.github.io: single-sign-on/index.md

To configure SSO, log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} to obtain the **ACS URL** and **Entity IDs** to complete the IdP server configuration process. You can only configure SSO with a single IdP.  When this is complete, log back into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and complete the SSO enablement process.

> **Important**
>
> If your IdP setup requires an Entity ID and the ACS URL, you must select the

> > SAML tab in the Authentication Method section. For example, if your Azure AD

Reviewer: ```suggestion
> **SAML** tab in the **Authentication Method** section. For example, if your Azure AD
```

Author: Updated

Author: Updated

> > setup uses SAML configuration within Azure AD, you must select SAML. If you

Reviewer: ```suggestion
> setup uses SAML configuration within Azure AD, you must select **SAML**. If you
```

> > are [configuring Open ID Connect with Azure AD](https://docs.microsoft.com/en-us/powerapps/maker/portals/configure/configure-openid-settings){:

Reviewer: We don't need to the CTA ID for the URL as we are not redirecting users to a Docker website

Author: Thanks Usha! Removing it

>  target="_blank" rel="noopener" class="_" id="dkr_docs_cta"}, select the Azure AD as the

Reviewer: ```suggestion
 target="_blank" rel="noopener" class="_" id="dkr_docs_cta"}, select the **Azure AD** as the
```

Author: Updated

7. Proceed to **add your domain** before you test and enforce SSO.

### Azure AD IdP configuration with Open ID Connect

>**Note:**
>
> This section is for users who only want to configure Open ID Connect with

> > Azure AD. This connection is not a full OpenID Connect with customization.

Reviewer 2: ```suggestion
> Azure AD. This connection is a basic OIDC connection and there are no special customizations available when using it.
```

Author: Thanks Josh, updated!

7. Proceed to **add your domain** before you test and enforce SSO.

### Azure AD IdP configuration with Open ID Connect

> 

Reviewer: Some users may land directly on this section. Can we also add the note here to convey that users must follow these instructions only when configuring Open ID Connect with Azure AD?

Author: Updated
